modules/EvseManager/Charger.cpp

```diff
@@ -64,7 +66,7 @@ void Charger::mainThread() {
             break;
         }
 
-        std::this_thread::sleep_for(std::chrono::milliseconds(50));
+        std::this_thread::sleep_for(50ms);
```

Comment:
I would prefer to have these as constants,
like static const auto SOME_CONSTANT_NAME = std::chrono::milliseconds(50);